packages/components/src/utils/generate-card.ts

```diff
@@ -1,16 +1,13 @@
-import { PlayingCard, PlayingCardName, PlayingCardSuit } from '@models/PlayingCard';
+import { allPlayingCardNames, allPlayingCardSuits, PlayingCard } from '@models/PlayingCard';
```

Comment:
No big deal and super minor, but I don't think we need the prefix `all` => `playingCardNames` would be enough in my opinion 😄 

Also, I like it better in camel case, so maybe you could also change it for `AllUnoColors` & values.

I will approve anyway 😄

Reply:
e.g. `PlayingCardNames` ?
I since it's a constant, wouldn't that break the linter in this case?

Anyhow, I am ok for doing it. Perhaps the only point that is a bit slugish is that we will have a type `PlayingCardName` and an array constant as `PlayingCardNames`.

Shall we move forward?